### PR TITLE
Public top banner section

### PR DESCRIPTION
**Pull request by thombudel**
screen shot 2017-03-28 at 12 35 05

### CODE REVIEW

**Comment by arnoFleming** [comment]
fix merge issues first!

**Comment by thombudel** [comment]
Removed container tag, because it will used in app.html. Added text align class.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -9,7 +9,11 @@
   </head>
 
   <body>
+
     <%= render './shared/navbar' %>
+    <%= render 'public/top_banner_section' %>
+
+
```
Comment (arnoFleming) [inline app/views/layouts/application.html.erb]: remove extraneous newlines

File: app/views/public/_top_banner_section.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+
```
Comment (arnoFleming) [inline app/views/public/_top_banner_section.html.erb]: idem

File: app/views/public/_top_banner_section.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
```
Comment (arnoFleming) [inline app/views/public/_top_banner_section.html.erb]: why is this in `public` and not in `shared`?

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -7,13 +7,9 @@
...
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
-
   <body>
-
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
     <%= render './shared/navbar' %>
-    
+    <%= render 'shared/top_banner_section' %>
```
Comment (miriamtocino) [inline app/views/layouts/application.html.erb]: The top banner section will be probably different in each of the pages. Also take into account that the homepage doesn't have a banner like the other 3 pages.

If that's the case, I don't think you want to render the banner in this file, but actually in each of the different pages where it belongs :wink:

Comment (thombudel) [inline app/views/layouts/application.html.erb]: Clear! I hope...

I have removed the render out of this file now.

**Pull request merged**